server.go

```diff
@@ -1640,8 +1645,24 @@ func (s *server) createLivenessMonitor(cfg *Config, cc *chainreg.ChainControl) {
 	tlsHealthCheck := healthcheck.NewObservation(
 		"tls",
 		func() error {
-			_, parsedCert, err := cert.LoadCert(
-				cfg.TLSCertPath, cfg.TLSKeyPath,
+			certBytes, keyBytes, err := cert.GetCertBytesFromPath(
```

Comment:
I think we could hide this in `TLSManager` as well, it would place the complexity into the more specialized struct.

Reply:
Good call